### DISCUSSION TITLE
Optimizing Image Consistency through Preset Settings

### DISCUSSION THREAD

**Discussion by martinmildner** [awcodes/filament-curator#139] [Ideas]
Currently, it is possible to select a preset and then adjust the settings (width, height) that have been defined in a preset and save it with the given preset key.

Wouldn't new image versions be more consistent across all other images if the values set by the presets were not adjustable once a preset has been selected? In this case, it would be nice to have the ability to add additional settings (quality, type, etc.) when defining a preset. What do you think?

**Comment by awcodes** [comment]
I do think adding the quality etc to defining presets could be good.

I can see the point of restricting the editing of settings when using a preset, but I'd still prefer if the user can also use presets as a starting point to create custom versions of the preset when they might be needed in one of situations. I also think people might be misusing or misunderstanding the curations a little too. They are really more for focal point adjustments, other wise it's better to use the glider component to output say 'thumbnails' via your blade file, since they are cached and can be cleared it keeps you from having a lot of unnecessary files just taking up space on your server and never being seen on the front end.

**Comment by martinmildner** [comment]
Okay, I guess I was too fixated on that other direction and use case ;-) This explanation makes perfect sense now, thanks @awcodes !
